charts/kubescape-operator/values.yaml

```diff
@@ -140,7 +141,7 @@ kubescape:
   image:
     # -- source code: https://github.com/kubescape/kubescape/tree/master/httphandler (public repo)
     repository: quay.io/kubescape/kubescape
-    tag: v2.9.2
+    tag: v2.9.3-prerelease
```

Comment:
@amirmalka  I think we should wait for the official release.

Reply:
whatever your decide.. the other components also will not be released without tests passing .. so..